Geyser.container:raise/lower changes order in Geyser.windows

Edru2 opened the pull request:
#### Brief overview of PR changes/additions
raising and lowering a window changes the index of the windowname in the containers container:window list.

#### Motivation for adding to Mudlet
This is needed because otherwise changes made by raise/lower are destroyed if someone uses raiseAll/lowerAll

#### Other info (issues closed, discussion etc)
There is still some discussion needed as for how to change the index.

For me it is more intuitive to change the index by +1 on raising and by -1 on lowering (and like that having a z-index) but the comment on container:raise or container:lower is suggesting that if raising a window it should put on top of the index and if lowering on the bottom.

In the end this only affects people who use raiseAll/lowerAll or Adjustable.Containers which have raiseAll activated by default (but it can be deactivated/is optional)

I hope this is somehow understandable.

Edru2 committed:
first commit

Geyser raise/lower keep z-index if raiseAll/lowerAll

vadi2 reviewed (commented):
Changing the Z index by +/-1 makes sense to me, as long as it doesn't change any behavior scripts already depend upon.

Edru2 commented:
After testing and reading the qt documentation I come to the conclusion that to be consistent it has to be top/bottom of the index.
Will change it.

If there is someone interested in it, it would be possible to make lowerByOne or raiseByOne functions (only for Geyser) but that would be for another PR.

Edru2 committed:
put window at top/bottom of index

To be consistent with the raise/lower functions the window has to be put on top or bottom of the index.
